documentation/sphinx/source/data-modeling.rst

```diff
@@ -269,6 +269,22 @@ Using the table name as the subspace, we could implement the common row-oriented
             cols[c] = v
         return cols
 
+
+Versionstamps
+-------------
+
+A common data model is to index your data with a sequencing prefix to allow log scans or tails of recent data. This index requires a unique, monotonically increasing value. Rather than implementing this feature yourself at the client level, which would require reading the value for conflict checks before every increment, the versionstamp provides unique, in-order id generation with a single conflict-free write.
```

Comment:
We may want to be more precise around what "conflict-free write" means. The write still generates a write conflict range, but we can generate a sequence ID with these properties without any read conflict ranges.

Reply:
Sounds good, I will go into more detail here